compairifuel/lib/src/global/ui/widgets/align_pop_up.dart

```diff
@@ -0,0 +1,44 @@
+import 'package:compairifuel/src/global/ui/widgets/pop_up.dart';
+import 'package:flutter/material.dart';
+
+class AlignPopUp extends StatelessWidget {
+  const AlignPopUp(
+      {super.key,
+      required this.padding,
+      required this.body,
+      required this.buttons,
+      this.offset,
+      this.size});
+
+  final EdgeInsets padding;
+  final WidgetBuilder body;
+  final List<Widget> buttons;
+  final Size? size;
+  final Offset? offset;
+
+  @override
+  Widget build(BuildContext context) {
+    Size popUpSize = size ?? MediaQuery.sizeOf(context);
+    Offset popUpOffset = offset ?? Offset.zero;
+
+    var bottomPadding =
+        MediaQuery.sizeOf(context).bottomCenter(popUpOffset).dy -
```

Comment:
bij het veranderen van de schermgrootte(verticaal) tijdens de popup krijg je een foutmelding dat de bottompadding negatief is.

Als je de schermgroote horizontaal veranderd (en de mode van sidenav naar bottonnav gaat) worden de popups op elkaar geplaatst. Dit is omdat er opnieuw de initialize gedaan wordt(vermoedelijk). Dat is ook waarop de locatie en size ook niet aangepast wordt als de grootte veranderd. (In de build van de map_screen gooien werkte niet)

Bottompadding en rightpadding beginnen allebei op 0 in fullscreen
In een kleiner scherm wordt bottompadding 61 en rightpadding 0, dat is goed

Reply:
Als aanvulling op het horizontaal veranderen van de schermgrootte:

![afbeelding](https://github.com/user-attachments/assets/a6170aaf-aed6-4bb8-951e-19f1e1c500a0)
Dit krijg ik te zien als ik de breedte van mijn window klein genoeg maak dat de navigatie en popup niet meer samen op een scherm past. Denk niet dat dit de bedoeling is. Het platform dat ik hiervoor gebruikt heb is windows.